comps/comps-foreman-client-sles155.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+<comps>
+  <!--  <meta> -->
+  <!-- Meta information will go here eventually -->
+  <!--  </meta> -->
+  <group>
+    <id>foreman-client</id>
+    <name>Foreman Client</name>
+    <description>Foreman Client Packages</description>
+    <uservisible>true</uservisible>
+    <packagelist>
+      <packagereq type="default">foreman-client-release</packagereq>
```

Comment:
Perhaps leave this out for now and make sure we only build it once we've updated it to contain correct .repo files?

Reply:
done